src/rqt_gauges_2/main.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+from rqt_gui.main import Main
+
+
+def main():
+    main = Main()
+    sys.exit(main.main(sys.argv, standalone='rqt_gauges_2.gauges_2.Gauges2'))
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
nit: missing empty line at the end of the file

Reply:
Done.